test/test_search.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+
+from tinygrad.codegen.linearizer import Linearizer
+from tinygrad.features.search import time_linearizer
+from tinygrad.ops import Compiled, Device, LoadOps
+from tinygrad.tensor import Tensor
+
+@unittest.skipIf(not isinstance(Device[Device.DEFAULT], Compiled), "only test for compiled backends")
```

Comment:
don't access device[device.default] in the decorator, it doesn't play well with some backends (eg ocelot) and causes very mysterious test failures. can just write this in setUp.

Reply:
changed for this one.
we use this pattern a lot tho - might need a codemod